tests/Aspire.Hosting.Tests/Dcp/ApplicationExecutorTests.cs

```diff
@@ -88,7 +88,7 @@ public async Task EndpointPortsExecutableNotReplicatedProxiedPortSetNoTargetPort
         Assert.True(dcpExe.TryGetAnnotationAsObjectList<ServiceProducerAnnotation>(CustomResource.ServiceProducerAnnotation, out var spAnnList));
```

Comment:
Is there any way to test this change?

Reply:
@radical discovered this issue by writing an end-to-end test with his workload infrastructure. That is not in yet, but he is aiming on pushing a PR up next week.

Reply:
Tests pass with the changes from this PR added to my workload-tests branch.